### ISSUE TITLE
Order Creation: Accessibility → Popup menu items should be marked as "Button"

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-ios#6068]
When using VoiceOver, these menu items should be dictated as "Button" instead of just the title and description. The "Button" further clarifies that they should select one of these to continue. 

<img src="https://user-images.githubusercontent.com/198826/152428356-bc6bbe94-182f-46d5-a950-e0e182712bda.png" width="300">
